samples/model-builder/create_and_import_dataset_tabular_bigquery_sample.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Union
+
+from google.cloud import aiplatform
+
+
+#  [START aiplatform_sdk_create_and_import_dataset_tabular_bigquery_sample]
+def create_and_import_dataset_tabular_bigquery_sample(
```

Comment:
Is this a dupe of this: https://github.com/googleapis/python-aiplatform/pull/338/files#diff-1efd7e79aab7a269e6ed5cb0ce9d4a9c3016782762156e3869a4d77d9d4824daR20

Reply:
Yes, but it's under my name. Let me check with @aribray.

Reply:
The assignments might have changed between her writing the sample and me starting. I will remove these and we can use Ari's.